components/automate-ui/src/app/components/pending-edits-bar/pending-edits-bar.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, OnInit, OnDestroy, ViewEncapsulation } from '@angular/core';
+import { interval as observableInterval, Subject } from 'rxjs';
+import { take, takeUntil } from 'rxjs/operators';
+import { Store } from '@ngrx/store';
+import { NgrxStateAtom } from 'app/ngrx.reducers';
+
+import { ProjectService } from 'app/entities/projects/project.service';
+import { Project } from 'app/entities/projects/project.model';
+import { allProjects } from 'app/entities/projects/project.selectors';
+
+@Component({
+  selector: 'app-pending-edits-bar',
+  templateUrl: './pending-edits-bar.component.html',
+  styleUrls: ['./pending-edits-bar.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class PendingEditsBarComponent implements OnInit, OnDestroy {
+  public showPendingEditsBar = false;
+  public confirmApplyStartModalVisible = false;
+  private isDestroyed = new Subject<boolean>();
+
+  constructor(
+    private store: Store<NgrxStateAtom>,
+    public projects: ProjectService
+    ) {
+      this.store.select(allProjects).pipe(
+        takeUntil(this.isDestroyed)
+      ).subscribe((projectList: Project[]) => {
+        this.showPendingEditsBar = projectList.some(p => p.status === 'EDITS_PENDING');
+      });
+    }
+
+  ngOnInit(): void {}
+
+  ngOnDestroy(): void {
+    this.isDestroyed.next(true);
+    this.isDestroyed.complete();
+  }
+
+  public openConfirmUpdateStartModal(): void {
+    this.confirmApplyStartModalVisible = true;
+  }
+
+  private closeConfirmApplyStartModal(): void {
+    this.confirmApplyStartModalVisible = false;
+  }
+
+  public confirmApplyStart(): void {
```

Comment:
Much of the code in this component and in `ProcessProgressBarComponent` migrated from `ProjectListComponent`. Probably in your cleanup to be done, but just a reminder that you still need to delete those bits from `ProjectListComponent`.